src/test/java/OsduSimpleClientTest.java

```diff
@@ -0,0 +1,59 @@
+import java.io.IOException;
+
+import org.json.JSONException;
+import org.testng.annotations.*;
+import pariveda.osdu.OsduSimpleClient;
+
+public class OsduSimpleClientTest {
+
+    private OsduSimpleClient simpleClient;
+    private String dataPartition;
+    private String token;
+    private String successUrl;
+    private String successContentType;
+ 
+    @BeforeClass
+    public void setUp() {
+      dataPartition = "opendes";
+      token = "abcde";
+      successUrl = "https://server.test.com";
+      successContentType = "application/json";
+      simpleClient = new OsduSimpleClient(successUrl, dataPartition, successContentType, token);
+    }
+    
+    @Test(groups = { "integration" }, enabled=false ) // Disabled for now; need to refactor to leverage env variables for valid tokens
+    public void expectedSearchSuccess() {
+      String query = "{ kind: \"*:*:*:*\", limit: 1 }";
+      try {
+        simpleClient.Search(query);
+        System.out.println("expectedSearchSuccess: Search query is valid.");
+      } catch (Exception e) {
+          org.testng.Assert.fail("unexpected exception", e);
+      }
+    }
+    
+    @Test(groups = { "validation" }, 
+    expectedExceptions = JSONException.class) 
+    public void expectedSearchJSONError() {
+        String query = "test123";
+        try {
+            simpleClient.Search(query);
```

Comment:
should this give some sort of assert of failure if no error is thrown?

Reply:
In line 36, there's a testng annotation for expectedExceptions. the test will fail if the specified exception type is not thrown at all (or if a different exception type is thrown instead). I can definitely add some comments to reinforce this though